tests/unit/core/utils/test_i18n.js

```diff
@@ -0,0 +1,76 @@
+// import url from 'url';
```

Comment:
I think you meant to remove all of this.

Reply:
ha yes ...sorry about that. I am rushing :/